lib/storext/class_methods.rb

```diff
@@ -23,7 +23,7 @@ def store_attribute(column, attr, type=nil, opts={})
       track_store_attribute(column, attr, type, opts)
       storext_check_attr_validity(attr, type, opts)
       storext_define_accessor(column, attr)
-      store_accessor column, *storext_attrs_for(column)
+      store_accessor(column, attr)
```

Comment:
Does this line solve an underlying bug? If so, then a test that replicates the bug ought to drive this code. 

The corresponding test in `spec/storext_spec.rb` does not ensure that there's an existing store accessor that has been prevented from being redefined. It only expected what's implemented.